config/initializers/date_formats.rb

```diff
@@ -5,3 +5,5 @@
 Time::DATE_FORMATS[:day_month_year] = "%d %B %Y"
 # For example: 6:53pm on 2 December 2019
 Time::DATE_FORMATS[:time_on_day_month_year] = "%l:%M%P on %e %B %Y"
+# For example: December 2019
+Time::DATE_FORMATS[:time_on_month_year] = "%B %Y"
```

Comment:
```suggestion
Time::DATE_FORMATS[:month_year] = "%B %Y"
```

Since there is no `time on` bit!